### DISCUSSION TITLE
v1.1.0.0 Release - Settings Menu

### DISCUSSION THREAD

**Discussion by RadzPrower** [ajohns6/Zelda-3-Launcher#2] [Announcements]
![image](https://user-images.githubusercontent.com/37708128/218199443-33368a42-0dbf-4b23-836e-073fe9f1e820.png)
The v1.1 release includes the addition of a settings menu that will allow you to manage settings for the game. All current settings are included other than keymapping (that will come in a v1.2 with controller recognition hopefully sometime next week).

**NOTE:** All features which can be edited by this tool are original to the game and a product of snesrev's work. I created a more friendly interface to manage those settings. I did not create any of these mods.

Bug fixes:
- Fixed the bug where the build process would loop infinitely if you canceled the ROM selection dialog.
- Fixed the bug where the launcher was not recognizing a zelda3.sfc file in the same directory as the launcher.

Features added:
- Progress bar for build/restore/update process in main launcher window.
- Minimize the launcher when launching the game and restore it to the front when you close it.
- A comprehensive settings menu that includes settings including aspect ratio, stretch, window scaling, fullscreen, disabling audio, MSU, custom Link sprites, and GLSL shaders.
- Also includes settings for a number of gameplay-altering features including an advanced item selection mode, turning while dashing, mirror works in both directions, larger wallet, and more.

<hr /><em>This discussion was created from the release <a href='https://github.com/ajohns6/Zelda-3-Launcher/releases/tag/'>v1.1.0.0 Release - Settings Menu</a>.</em>

**Comment by Maverick0741** [comment]
How do you set this up for Steam Deck? I haven't had any luck using Proton so far

**Reply by RadzPrower** [reply]
Don't use Proton or this launcher.

Build a Linux native version either on a Linux machine or something like WSL or a VM on Windows and copy it over to your Deck.

**Reply by Maverick0741** [reply]
Since the Steam Deck is a Linux machine, how would I build the native linux version on that?

**Reply by RadzPrower** [reply]
You don't.

Steam Deck is not meant to be a building machine. Valve has removed a number of necessary dependencies and you would have to do things that are very bad for your device security to even try to build it directly on the Steam Deck.

**Reply by Formedras** [reply]
This might be valuable: https://github.com/mahboi/zelda3-flatpak

**Comment by MrYama** [comment]
I can't get this too work, always says that it can't execute due to a lack of tables

**Reply by RadzPrower** [reply]
Well, this isn't even the latest release, so that may be you're problem. Latest release right now is here: https://github.com/RadzPrower/Zelda-3-Launcher/releases/tag/v1.3.6.0
